### DISCUSSION TITLE
UK energy price cap rate?

### DISCUSSION THREAD

**Discussion by jrb80** [BottlecapDave/HomeAssistant-OctopusEnergy#571] [General]
I want to plot a graph of my Agile rates against the [UK energy price cap rate](https://www.ofgem.gov.uk/energy-price-cap), currently 27.35 pence per kWh.

I could create a sensor template and manually input this value but I was wondering whether anyone knew of a API data source where I could pull it?

**Comment by BottlecapDave** [comment]
I couldn't find any online sources, which is why it's manual in the integration. I believe on some tariffs, OE apply the cap to the data returned by the API, but some tariffs they don't (hense the override).

**Comment by pcb1962** [comment]
> the [UK energy price cap rate](https://www.ofgem.gov.uk/energy-price-cap), currently 27.35 pence per kWh

Worth pointing out that if you're on Agile the cap is 100p so you don't want to put the 27.35p value into your configuration (like I just did!)

**Comment by jrb80** [comment]
It's mainly to do a visual comparison against the UK energy price cap to understand how often it is below/above to see whether it makes sense to stay on the Agile tariff.

![tempFileForShare_20231126-085855](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/assets/1820167/025ddf63-334a-467b-8906-d80ee00b33f7)

**Reply by BottlecapDave** [reply]
If you're wanting to compare another tariff, you can use [tariff overrides](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/blob/develop/_docs/entities/electricity.md#tariff-overrides) as another option. I use this to roughly compare my current tariff and another fixed tariff, which I plot on a chart.

**Comment by jrb80** [comment]
> If you're wanting to compare another tariff, you can use [tariff overrides](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/blob/develop/_docs/entities/electricity.md#tariff-overrides) 

Oh, nice. Thanks for the tip. I didn't know you could do that. The Octopus API returns 40 products and it's a bit tricky to tell which one is aligned with the UK energy cap, however I think it's [VAR-22-11-01](https://api.octopus.energy/v1/products/VAR-22-11-01/)
